components/DownloadDataset/DownloadDataset.vue

```diff
@@ -53,22 +56,16 @@
         <div class="text-block aws">
           us-east-1
         </div>
-        <div class="buttons">
-          <nuxt-link
-            :to="{
-              name: 'help-helpId',
-              params: {
-                helpId: helpId
-              }
-            }"
-          >
-            <bf-button class="download-button">
-              More Information
-            </bf-button>
-          </nuxt-link>
-          <bf-button class="secondary button-spacing" @click="closeDialog">
-            Okay
+        <div class="disclosure-text-block">
+          <p>*Requester pays means that any costs associated with downloading the data will be charged to your AWS account.
+            For transfer pricing information, visit the <a href="https://aws.amazon.com/s3/pricing/" target="blank">AWS Pricing documentation.</a>
+          </p>
+          <div>
+            <bf-button class="secondary button-spacing" @click="closeDialog">
```

Comment:
`bf-button` has been deprecated. Can this be replaced with `el-button`? I know there was a ticket to make all of the buttons consistent, and the solve was to use `el-button`, but I know that ticket hasn't been done yet (in favor of just using the design system.)
```suggestion
            <el-button class="secondary" @click="closeDialog">
```